src/peergos/server/sql/SqliteCommands.java

```diff
@@ -0,0 +1,15 @@
+package peergos.server.sql;
+
+public class SqliteCommands implements SqlSupplier {
```

Comment:
As discussed I think we should  look to  remove any DDL  from these interfaces/implementations, which amounts to removing the `SqlSupplier` interface I think. We can do this in a  separate PR.